README.md

```diff
@@ -1,3 +1,31 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
awesome

Reply:
Pls add a link to `doctoc` + a line about how to install/run